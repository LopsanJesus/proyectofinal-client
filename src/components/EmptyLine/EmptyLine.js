import React from "react";
import { Col, Form, Row } from "react-bootstrap";
import "./EmptyLine.scss";

const EmptyLine = ({ number }) => {
  return (
    <Row key={number}>
      <Col>
        <Form.Control type="text" required />
      </Col>
      <Col>
        <Form.Control type="text" required />
      </Col>
    </Row>
  );
};

export default EmptyLine;
