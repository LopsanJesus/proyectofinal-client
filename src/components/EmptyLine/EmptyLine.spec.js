import React from "react";
import { shallow } from "enzyme";
import EmptyLine from "./EmptyLine";

describe("EmptyLine", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<EmptyLine />);
    expect(wrapper).toMatchSnapshot();
  });
});
