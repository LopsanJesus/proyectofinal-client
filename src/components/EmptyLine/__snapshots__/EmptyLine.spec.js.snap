// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyLine matches snapshot 1`] = `ShallowWrapper {}`;
